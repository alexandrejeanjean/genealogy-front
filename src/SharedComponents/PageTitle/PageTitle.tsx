import React from 'react'

type Props = {
  pageTitle?: string
}

const PageTitle = ({ pageTitle }: Props) =>
  pageTitle ? (
    <div className='title-wrapper'>
      <h1>{pageTitle}</h1>
    </div>
  ) : null

export default PageTitle
