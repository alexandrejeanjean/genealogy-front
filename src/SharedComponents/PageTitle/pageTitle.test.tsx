import React from "react";
import { mount } from "enzyme";
import PageTitle from "./PageTitle";

describe("PageTitle", () => {
  it("should render page title if pageTitle props exist and check if props is equal to Dashboard", () => {
    const Component = mount(<PageTitle pageTitle="Dashboard" />);
    expect(Component.props().pageTitle).toEqual("Dashboard");
    expect(Component).toMatchSnapshot();
  });
});
