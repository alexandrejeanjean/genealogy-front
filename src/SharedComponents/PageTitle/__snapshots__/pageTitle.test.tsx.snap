// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle should render page title if pageTitle props exist and check if props is equal to Dashboard 1`] = `
<PageTitle
  pageTitle="Dashboard"
>
  <div
    className="title-wrapper"
  >
    <h1>
      Dashboard
    </h1>
  </div>
</PageTitle>
`;
